src/config.c

```diff
@@ -250,6 +244,9 @@ static int config_parse_command (struct configuration *config,
     } else if (match(command, "verbosity")) {
         parse_check(parse_int((unsigned int*)&config->verbosity, arg, num, 0, 5));
 
+    } else if (match(command, "threads")) {
```

Comment:
There are a handful of files that need copyrights updated.

Reply:
updated